cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -6,23 +6,23 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
+// The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
+
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
```

Comment:
You tell us the benefit of installing the OpenShift VS Code extension: it uses both the odo and oc binaries and interacts with the user's OpenShift Container Platform cluster. 

So, what's the benefit of installing the odo CLI as an RPM? If you give users multiple options, give them a benefit for each option (like you did for the OpenShift VS Code extension). Basically, right now, I don't know why I would want/consider installing the odo cli as an RPM. 

Also, tell us what RPM is, then put the acronym in parenthesis since this is our first time seeing the acronym.